src/content/docs/en/install/auto.mdx

```diff
@@ -1,154 +1,420 @@
 ---
-title: Install Astro with the Automatic CLI
-description: 'How to install Astro with NPM, PNPM, or Yarn via the create-astro CLI tool.'
-i18nReady: true
+title: Install Astro
+description: A reference page in my new Starlight docs site.
 ---
-import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Tabs, TabItem, FileTree, CardGrid, LinkCard, Steps } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
 
+The [`create astro` CLI command](#install-from-the-cli-wizard) is the fastest way to start a new Astro project from scratch. It will walk you through every step of setting up your new Astro project and allows you to choose from a few different official starter templates. 
 
-Ready to install Astro? Follow this guide to using the `create astro` CLI to get started.
+Or, you can begin your project [using any existing theme or starter template](#use-a-theme-or-starter-template).
 
-:::note[Prefer to install Astro manually?]
-Read our [step-by-step manual installation guide](/en/install/manual/) instead.
+To install manually instead, see our [step-by-step manual installation guide](#manual-setup).
```

Comment:
```suggestion
To install Astro manually see our [step-by-step manual installation guide](#manual-setup).
```

Reply:
The "instead" is important here, but I do like including "Astro"! Thank you!